src/libraries/System.Runtime.InteropServices/tests/Directory.Build.props

```diff
@@ -0,0 +1,11 @@
+﻿<Project>
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+    <IsShipping>false</IsShipping>
+    <IsPackable>false</IsPackable>
+    <!-- We manually enable DllImportGenerator for projects in this folder as part of testing. -->
+    <EnableDllImportGenerator>false</EnableDllImportGenerator>
```

Comment:
Why do we need to explicitly set these? I thought these would the default for test projects under tests/?